src/classes/ffhttp_ClientTest.cls

```diff
@@ -382,6 +382,24 @@ private class ffhttp_ClientTest
 		System.assert(response instanceof JsonTest, 'Response should be an instance of JsonTest');
 		System.assertEquals(((JsonTest) response).getTest(), 'Test');
 	}
+	
+	/**
++	 * Check that a request with a Blob response is dealt with appropriately.
++	 */
++	static testMethod void testExecuteValidBlobResponse()
```

Comment:
Use up to date code style even if this one is using the old stlyle:

@IsTest
private static void ....